src/lib/components/navbar.svelte

```diff
@@ -21,7 +21,7 @@
     },
     {
       title: 'Tutorial',
-      href: 'https://github.com/mermaid-js/mermaid/blob/develop/docs/Tutorials.md'
+      href: 'https://mermaid-js.github.io/mermaid/#/Tutorials'
```

Comment:
```suggestion
      href: 'https://mermaid-js.github.io/mermaid/config/Tutorials.html'
```

This will be the right URL once the new docs are released. 
I accidentally merged #1114, which was a duplicate of this PR. 

Will keep this open to make sure the correct link is used.